### PR TITLE
new pull request to add michelange

### PR DESCRIPTION
**Pull request by mickey0116**

### CODE REVIEW

File: about/people.yml [hunk 1]
```diff
@@ -465,12 +465,12 @@ data:
     brown_directory_uuid: 294d8182-b793-46ad-a6a8-6548b7378c2e
     bio: Sanjana is a data science intern working with the Computational Biology Core at CCV. She is concentrating in Computational Biology ('26) and is interested in genetics and public health. She also loves to read sci-fi, run, and spend time with friends!
     image: sanjana_main.jpg
-  - name: Jesus Rodriguez
-    type: Part Time
+  - name: Michelange Senat
```
Comment (eldu) [inline about/people.yml:468]: Looks like you've replaced Jesus's entry with yours! You'll need to keep his entry so copy paste it back in.

File: about/people.yml [hunk 2]
```diff
@@ -465,12 +465,12 @@ data:
...
+    type: Intern
```
Comment (eldu) [inline about/people.yml:469]:
```suggestion
    type: Part Time
```

File: about/people.yml [hunk 3]
```diff
@@ -465,12 +465,12 @@ data:
...
-    title: Data Science Intern
-    github_username: monoguitari
-    brown_directory_uuid: 6be48c4b-0881-457b-b3f8-b0376b589fdb
-    bio: Jesus is a data science intern working with the High Performance Computing division at CCV. He is a rising junior at Brown University studying Computer Science. His academic interests include software development, machine learning, and mathematics. Outside of work, you can find him playing guitar, learning Vietnamese, or listening to music.
-    image: jesus_main.jpg
+    subteam: Graphics, Software, and Data Core
+    title: Software Sustainability Intern
+    github_username: mickey0116
+    brown_directory_uuid: 39f28676-e6ec-4ced-afc3-1b3e5d70683c
+    bio: Michelange is a summer intern wotking with the Graphics, Software, and Data Core at CCV. She is an undergraduate student at Brown Univeristy interested in studying Applied Mathematics, Computer Science, and Econommics. Outside of academics, Michelange loves matcha, hair braiding, and exploring different neighborhoods in Rhode Island.
```
Comment (eldu) [inline about/people.yml:475]: typos
```suggestion
    bio: Michelange is a summer intern working with the Graphics, Software, and Data Core at CCV. She is an undergraduate student at Brown University interested in studying Applied Mathematics, Computer Science, and Economics. Outside of academics, Michelange loves matcha, hair braiding, and exploring different neighborhoods in Rhode Island. 
```

**Review by eldu — CHANGES_REQUESTED** [review]
The format of the yaml looks good and the images too! I made some comments on typos and work changes. Looks like you've replaced Jesus! So let's bring him back into the website. 

Looks good so far though !

**Review by mickey0116 — COMMENTED** [review]
added Jesus back and fixed my errors

File: about/people.yml [hunk 4]
```diff
@@ -464,7 +464,7 @@ data:
     github_username: sanjana-hiremath
     brown_directory_uuid: 294d8182-b793-46ad-a6a8-6548b7378c2e
     bio: Sanjana is a data science intern working with the Computational Biology Core at CCV. She is concentrating in Computational Biology ('26) and is interested in genetics and public health. She also loves to read sci-fi, run, and spend time with friends!
-    image: sanjana_main.jpg
+    image: sanjana_main.jp
```
Comment (broarr) [inline about/people.yml:467]: It looks like this might be a typo. Can you change this back to `jpg` please!

**Review by broarr — CHANGES_REQUESTED** [review]

**Comment by broarr** [comment]
Looks great after your revisions! One little typo and I think we're good to merge!

**Comment by broarr** [comment]
You'll have to merge main in and resolve the conflicts. Let me know if you'd like help with that

**Review by broarr — APPROVED** [review]
Looks great, thank you! 🎉

**Review by mickey0116 — COMMENTED** [review]
done!

**Review by eldu — APPROVED** [review]
Amazing!!! Merge it in when you get the chance :)

**Pull request merged**